x/epochstorage/keeper/stake_storage.go

```diff
@@ -137,10 +137,11 @@ func (k Keeper) removeAllEntriesPriorToBlockNumber(ctx sdk.Context, block uint64
 						// if storageBlock is empty its stake entry current. so we dont remove it.
 						continue
 					}
-					// panic:ok: should never happen; avoid further storage corruption
-					utils.LavaFormatPanic("critical: failed to decode storage block", err,
+					utils.LavaFormatError("critical: failed to decode storage block", err,
+						utils.LogAttr("chainID", chainId),
 						utils.LogAttr("index", entry.Index),
 						utils.LogAttr("storageBlock", storageBlock),
+						utils.LogAttr("block", block),
 					)
 				}
 				if blockHeight < block {
```

Comment:
the parsing code panics if it is asked to deal with unknown type (.e. anything not uint64), but returns 0 (and an error) if the parsing itself fails.  @Yaroms is correct to note that we decided to let it be because it's only called from testing code.